src/styles/card.scss

```diff
@@ -0,0 +1,111 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
```

Comment:
Default value
```suggestion
```

Reply:
Is it possible not to use it at all? Should it be added to the variables?